packages/medusa/src/loaders/plugins.ts

```diff
@@ -83,6 +83,14 @@ export default async ({
     )
   )
 
+  // Adds core api routes
+  await new RoutesLoader({
+    app,
+    rootDir: path.join(__dirname, "../api-v2"),
```

Comment:
question: instead of introducing a new v2 folder, could we point directly to the new paths here as @kasperkristensen hinted?

Reply:
you mean something like this? If not, I'm gonna need some guidance here :D 

```
const domainPaths = ["admin/promotions"]

for (domainPath of domainPaths) {
  await new RoutesLoader({
    app,
    rootDir: path.join(__dirname, `../api/routes/${domainPath}`),
    activityId: activityId,
    configModule,
  }).load()
}
```

Reply:
yeah something like this!

Reply:
moved it around, take a look